www/_posts/2024/2024-03-03-pyladies-at-pycon.md

```diff
@@ -0,0 +1,100 @@
+
+- [PyO3 101 - Writing Python modules in Rust](https://us.pycon.org/2024/schedule/presentation/113/) by [Cheuk Ting Ho](https://us.pycon.org/2024/speaker/profile/97/) - Room 319 
+
+
+**1:30pm - 5pm**
+
+- [Introduction to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more, from the comfort of your Python repl.](https://us.pycon.org/2024/schedule/presentation/55/) by [Naty Clementi](https://us.pycon.org/2024/speaker/profile/66/), [Gil Forsyth](https://us.pycon.org/2024/speaker/profile/64/) and [Phillip Cloud](https://us.pycon.org/2024/speaker/profile/65/) - Room 319
+
+- [Introduction to MicroPython: getting started with BBC micro:bit](https://us.pycon.org/2024/schedule/presentation/4/)  by [Juliana Karoline de Sousa](https://us.pycon.org/2024/speaker/profile/5/) - Room 310/311
+
+
+### Thursday 05/16
+
+**9am - 12:30pm**
+
+- [Monitoring carbon emissions: making the impact of your python code visible](https://us.pycon.org/2024/schedule/presentation/38/) by [Jessica Greene](https://us.pycon.org/2024/speaker/profile/44/) and [Chioma Onyekpere](https://us.pycon.org/2024/speaker/profile/45/) - Room 318
+
+
+
+## Talks and Charlas
+
+### Friday 05/17
+
+**12:30pm - 1pm**
+
+- [The Design of Everyday APIs](https://us.pycon.org/2024/schedule/presentation/25/) by [Lynn Root](https://us.pycon.org/2024/speaker/profile/31/) - Room 301-305
+
+- [Personalizando LLMs: Guía para “Fine-Tuning” local de modelos Open Source en Español](https://us.pycon.org/2024/schedule/presentation/99/) by [Maria Jose Molina Contreras](https://us.pycon.org/2024/speaker/profile/114/)  - Room 310/311
+
+
+**3:30pm - 4pm**
+
+- [Porque SQL está en todas partes.](https://us.pycon.org/2024/schedule/presentation/117/) by [Naty Clementi](https://us.pycon.org/2024/speaker/profile/66/) and [Phillip Cloud](https://us.pycon.org/2024/speaker/profile/65/) - Room 310/311
+
+
+**4:30pm - 5pm**
+
+- [How to amplify diversity inside a Python community?](https://us.pycon.org/2024/schedule/presentation/104/) by [Juliana (Jules) Barros Lima](https://us.pycon.org/2024/speaker/profile/122/) - Hall C
+
+
+**5:15pm - 5:45pm**
+
+- [¡Pero si funciona en mi computadora! Cómo lograr que nuestros análisis de datos sean reproducibles](https://us.pycon.org/2024/schedule/presentation/68/) by [Riva Quiroga](https://us.pycon.org/2024/speaker/profile/80/) - Room 310/311
+
+- [Acknowledging Women’s Contributions in the Python Community Through Podcast](https://us.pycon.org/2024/schedule/presentation/94/) by [Cheuk Ting Ho](https://us.pycon.org/2024/speaker/profile/97/), [Georgi Ker](https://us.pycon.org/2024/speaker/profile/109/), [Mariatta](https://us.pycon.org/2024/speaker/profile/108/), and [Tereza Iofciu](https://us.pycon.org/2024/speaker/profile/107/)
+
+### Saturday 05/18
+
+**10:30am - 11am**
+
+- [Observabilidad en Python para principiantes](https://us.pycon.org/2024/schedule/presentation/21/) by [Ana Paula Gonzaga](https://us.pycon.org/2024/speaker/profile/27/) - Room 310/311
+
```

Comment:
my pycon talk is 

https://us.pycon.org/2024/schedule/presentation/34/ may 18 10:30 in Hall C